src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractCreatePdpFeedServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpFeedService;
+import edu.cornell.kfs.concur.service.ConcurRequestExtractFileService;
+
+public class ConcurRequestExtractCreatePdpFeedServiceImpl implements ConcurRequestExtractCreatePdpFeedService {
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpFeedServiceImpl.class);
+	protected ConcurRequestExtractFileService concurRequestExtractFileService;
+
+	@Override
+    public void createPdpFeedsFromRequestExtracts() {
+        /* Get list of all Request Extract files that have not been processed into PDP Import files
```

Comment:
Removing pseudo code and replacing with method stubs. Please keep in mind, with this approach, method names and signatures most likely will need to be adjusted as the yet to be written implementation evolves.